src/Rust/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "easy_microservice"
```

Comment:
Is the easy_microservice packages name?

Reply:
yes, it will be used like this

`
use easy_microservice::file_manager;
`

Reply:
Ok, do we need to change it for now?

Reply:
There is no need to change it.
Also in crates.io we see no easy_microservice package , that means we can push it as a crate and reserve it for future uses.